odkx-src/sync-endpoint.rst

```diff
@@ -24,6 +24,8 @@ ODK-X Sync Endpoint
 It communicates with your ODK-X Android applications to synchronize
 your data and application files.
 
+There is an important library called :dfn:`Sync Client` which implements the sync protocol by interacting with Sync endpoint and other tools. The source of this library can be found here: `https://github.com/odk-x/sync-client <https://github.com/odk-x/sync-client/>`_. This is a libary that can be used to execute the ODK 2.0 sync protocol natively on non-Android systems.
```

Comment:
Endpoint is usually capitalized. 
Refer to ODK-X sync protocol not ODK 2.0.

Reply:
I fixed the spelling error, resolving this also.

Reply:
@elmps2018 Done!